src/ssl.c

```diff
@@ -19774,6 +19774,7 @@ void wolfSSL_FreeSession(WOLFSSL_CTX* ctx, WOLFSSL_SESSION* session)
 #ifdef HAVE_SESSION_TICKET
     if (session->ticketLenAlloc > 0) {
         XFREE(session->ticket, session->heap, DYNAMIC_TYPE_SESSION_TICK);
+        session->ticketLenAlloc = 0;
```

Comment:
Let's also clear `session->ticket`.

Reply:
Also `session->ticketNonce.data` is not being cleared. Please clear that too.

Reply:
done, i hope

Reply:
Great work!